CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+This guide provides instructions for contributing to this Capacitor plugin.
+
+## Developing
+
+### Local Setup
+
+1. Fork and clone the repo.
+1. Install the dependencies.
+
+    ```shell
+    npm install
+    ```
+
+1. Install SwiftLint if you're on macOS.
```

Comment:
Is there a point to add references to Swift in a plugin that does only work on Android?

Reply:
Hi this file is autogenerated I removed it.